agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/PreAggregatedMetricsTest.java

```diff
@@ -139,7 +139,7 @@ void generateHttpClientMetrics() {
     MetricTelemetryBuilder builder = MetricTelemetryBuilder.create();
     MetricData metricData = metricDataCollection.iterator().next();
     MetricDataMapper.updateMetricPointBuilder(
-        builder, metricData, metricData.getData().getPoints().iterator().next(), true, true);
+        builder, metricData, metricData.getData().getPoints().iterator().next(), true, true, null);
```

Comment:
metricData.getData() can never return null?

Reply:
(this is in test code)

Reply:
Maybe an assertion checking that metricData.getData() is not null could be useful in test code to have an explicit error message

Reply:
getData never returns null